### ISSUE TITLE
LINUX LEARNING ORDER

### ISSUE THREAD

**Issue by mohammedtabish** [ronreiter/interactive-tutorials#459]
In linux shell scripting the order of array comparison should come after learning for loops. Please change the order or provide referral to the loops section.

**Comment by stevula** [comment]
Agreed. The solution uses for loops but those haven't been taught at this stage.
